Packs/GoogleKubernetesEngine/Integrations/GoogleKubernetesEngine/GoogleKubernetesEngine.py

```diff
@@ -302,10 +301,10 @@ def gcloud_clusters_list_command(client: ClusterManagerClient, project: str, zon
         dict: Cluster raw response.
     """
     # Query and gPRC unpack
-    raw_response_msg: Message = client.list_clusters(project_id=project,
-                                                     zone=zone,
-                                                     timeout=API_TIMEOUT)
-    raw_response_dict: dict = MessageToDict(raw_response_msg._pb)  # type: ignore[attr-defined]
+    raw_response_msg = client.list_clusters(project_id=project,
```

Comment:
why did you remove the typing?

Reply:
wrong type, the type is `ListClustersResponse`